### PR TITLE
Add clearErrors parameter to ValidateMultiple

### PR DESCRIPTION
**Pull request by developeruz**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | yes
| Breaks BC?    | probably yes, because method signature is changed
| Tests pass?   | yes
| Fixed issues  | #9727

### CODE REVIEW

**Comment by samdark** [comment]
Should be pull requested against 2.1 since it's method signature change.

**Comment by developeruz** [comment]
Changed to be against 2.1

**Review by samdark — APPROVED** [review]

File: framework/base/Model.php [hunk 1]
```diff
@@ -887,17 +887,18 @@ public static function loadMultiple($models, $data, $formName = null)
      * be of the same or different types.
      * @param array $models the models to be validated
      * @param array $attributeNames list of attribute names that should be validated.
+     * @param bool $clearErrors whether to call [[clearErrors()]] before performing validation. Default true
```
Comment (MysteryDragon) [inline framework/base/Model.php]: This line must be moved after full `$attributeNames` description.

**Review by MysteryDragon — CHANGES_REQUESTED** [review]

**Review by MysteryDragon — APPROVED** [review]

**Comment by samdark** [comment]
Merged. Thank you!

**Pull request merged**